packages/neo-one-server-plugin-project/src/types.ts

```diff
@@ -30,13 +30,17 @@ export interface NEOTrackerRequestOptions {
 
 export type RequestOptions = NetworkRequestOptions | SourceMapsRequestOptions | NEOTrackerRequestOptions;
 
+export type CodegenLanguage = 'javascript' | 'typescript';
+export type CodegenFramework = 'none' | 'react' | 'angular' | 'vue';
```

Comment:
Remove angular and vue, they aren't possible in this PR.

Reply:
👍